### ISSUE TITLE
Can't start the game with bot as broadcaster

### ISSUE THREAD

**Issue by 7YZA1owXxM** [Arcana/node-dota2#545]
Seems like bot cannot connect to the server. Any way to fix this?

**Comment by angel200sdnot** [comment]
Start the game with bot in unassigned slot. The bot will keep getting the lobby updates in unassigned slot even when the game started.

**Comment by 7YZA1owXxM** [comment]
But broadcaster is needed for fetching live gamedata

**Comment by angel200sdnot** [comment]
Bots are not able to join the game as player or caster, the only live gamedata you can get is the one you get in practiceLobbyUpdate, which should be enough if you only want the heroes being played or who won the match.

See this issue for more information #240

**Comment by 7YZA1owXxM** [comment]
If i understand correctly, it is not literally bot. We're using the same login method and "speaking" with Steam / GC as common user , i.e. lib emulates steam/dota clients. So going further, i think there's a way to emulate connection to the game server...
Or not?

**Comment by Crazy-Duck** [comment]
Node-dota2 in the strictest sense is a re-engineering of the communication between the game client and steam + the game coordinator. This means we do *not* offer the possibility to communicate with the Valve Game Servers, as those use an entirely different protocol and format for communication, namely steam datagram, where GC communication is just protobuf (not to mention they're way pickier in terms of timing etc). 

In short, you can have your bot join a lobby and sit in a spectator slot where it will receive practiceLobbyUpdate events, but you cannot interact with the game server.

**Comment by 7YZA1owXxM** [comment]
Sad story, guys :(
Anyway, thanks for help!
